manifests/s/Sigil-Ebook/PageEdit/2.1.0/Sigil-Ebook.PageEdit.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=QUSU.CRLF.5-1-26100-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Sigil-Ebook.PageEdit
+PackageVersion: 2.1.0
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Upgrade: /NORESTART /SUPPRESSMSGBOXES /COMPONENTS="dicon,*afiles"
+UpgradeBehavior: install
+FileExtensions:
+- epub
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/Sigil-Ebook/PageEdit/releases/download/2.1.0/PageEdit-2.1.0-Windows-x64-Setup.exe
+  InstallerSha256: 6141FB96F5428093CD29CCD38BBA1C131A96D6CD47BB95C7BD529BAFCD93D016
+  InstallerSwitches:
+    Custom: /CURRENTUSER /NORESTART /SUPPRESSMSGBOXES /COMPONENTS="dicon,*afiles"
```

Comment:
```suggestion
```
Based on [mdanish-kh's Issue](https://github.com/microsoft/winget-cli/issues/4075), I believe this might just be a scope mismatch. `VCRedist` only has 1 scope and arch: `machine` `x64`

Reply:
I don’t want to just remove the user scope to "make it work". I'm hoping @denelon and the engineering team are able to take a look at the underlying behavior and either
1) Say it can be waivered, as this will only be an issue when there is a scope Requirement (`--scope` switch or user setting) and will not occur when scope is only a preference or is undefined

or

2) Make a change in CLI which resolves this (though I suspect CLI is funtioning as intended, just that pipelines now specify `--scope` when either they didn’t previously or it was ignored for dependencies)